app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -408,21 +429,23 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
             Tangram.addDataSource(searchResults);
         }
 
-        var activeFeature: Int = 0
+        var featureCount: Int = 0
         searchResults?.clear()
         for (feature in features) {
             val simpleFeature = SimpleFeature.fromFeature(feature)
             val lngLat = LngLat(simpleFeature.lng(), simpleFeature.lat())
             val properties = com.mapzen.tangram.Properties()
+            properties.add("tap", "search")
```

Comment:
We should consider adding constants for property values like `"tap"`, `"search"`, "`type`", and `"point"` so that we are not reusing the same string literals in several places.

Should these constants live in the app or does it make sense for some commonly used default values to be declared in Tangram?